docs/CloudPermissions.md

```diff
@@ -0,0 +1,56 @@
+# Restricting cloud privileges for deploying Scalar DL
+
+This guide explains how to restrict privileges for deploying Scalar DL on Azure.
+
+In general from security perspective, it is better to assign restricted privileges to users not to make them able to do operations that they are not supposed to do.
+
+## Azure
+
+The following JSON is a custom role that allow users to manage resources for Scalar DL. This is sufficient but not necessary.
```

Comment:
Sorry, What you mean by `This is sufficient but not necessary`?
Can we create without these permissions?

Reply:
>What you mean by This is sufficient but not necessary?

I meant this is sufficient permission but it's not necessary that the user should follow this custom role they can use an escalated permission if they want such as the `Contributor` role. Updated sentence as `These are the minimum permissions for creating a Scalar DL environment on the Azure cloud.` in c601da6